frontends/p4/typeChecking/typeUnification.cpp

```diff
@@ -46,8 +46,7 @@ bool TypeUnification::unifyCall(const IR::Node* errorPosition,
         if (dest->typeParameters->size() != src->typeArguments->size()) {
             TypeInference::typeError(
                 "%1% has %2% type parameters, but is invoked with %3% type arguments",
-                errorPosition, (dest->typeParameters ? dest->typeParameters->size() : 0),
-                src->typeArguments->size());
+                errorPosition, dest->typeParameters, src->typeArguments->size());
```

Comment:
This seems to be different in semantics; you are showing the arguments rather than their count.

Reply:
yes -- trivial typo in the change